stdin-stdout/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "jsonrpc-stdin-stdout-server"
+description = "STDIN/STDOUT server for JSON-RPC"
+version = "8.0.1"
```

Comment:
0.1 would be better starting version?

Reply:
Ah yeah, missed that on copy/paste. I have amended this PR with your remarks.